easybuild/easyconfigs/n/NAMD/NAMD-2.12-drop_space_after_backslash.patch

```diff
@@ -0,0 +1,12 @@
+diff -ru NAMD_2.12_Source.orig/src/ComputeNonbondedMICKernelBase2_handcode_single.h NAMD_2.12_Source/src/ComputeNonbondedMICKernelBase2_handcode_single.h
```

Comment:
@akesandgren please include a comment above to explain what this does, and where the patch comes from (author or reference)

Reply:
Comment and author has been added